src/main/java/com/spotify/reaper/cassandra/JmxConnectionFactory.java

```diff
@@ -51,7 +59,20 @@ public final JmxProxy connectAny(Optional<RepairStatusHandler> handler, Collecti
     if (hosts == null || hosts.isEmpty()) {
       throw new ReaperException("no hosts given for connectAny");
     }
-    return connect(handler, hosts.iterator().next());
+    List<String> hostList = new ArrayList<String>(hosts);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2293)